src/tests/evm_test.ts

```diff
@@ -84,11 +105,29 @@ test('Contract Calls', async t => {
       }
     }
 
-    const retVal = await evmContract.staticCallAsync(inputGetArray)
+    if (rtv) {
+      let rProto = await client.getTxReceiptAsync(rtv)
+      if (rProto) {
+        let receipt = EvmTxReceipt.deserializeBinary(bufferToProtobufBytes(rProto))
+        t.deepEqual(
+          receipt.getContractaddress_asU8().slice(),
+          contractAddr.local.bytes,
+          'Contract address should match'
+        )
+        t.equal(receipt.getStatus(), 1, 'Should return status 1 success')
+      } else {
+        t.fail('getTxReceiptAsync should return a result')
+      }
+    }
 
-    const retArr = Array.prototype.slice.call(retVal, 0)
-    const expectedRtv = inputSet987Array.slice(-32)
-    t.deepEqual(retArr, expectedRtv, 'Query result must match previously set')
+    const staticCallRtv = await evmContract.staticCallAsync(inputGetArray)
+    if (staticCallRtv) {
+      const retArr = Array.prototype.slice.call(staticCallRtv, 0)
```

Comment:
One more slice here.

Reply:
I could not get staticCallRtv.slice() to work. I'll have another go.

Reply:
So expectedRtv is an Array(32) while staticCallRtv.slice(0) is a Buffer(32) I always get
```
    operator: deepEqual
    expected: |-
      [ 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 3, 219 ]
    actual: |-
      <Buffer 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 03 db>
```